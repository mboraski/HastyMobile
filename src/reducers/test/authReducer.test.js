// TODO: Remove when tests implemented
test.only('real recognize real', () => {
    expect('real').toBe('real');
});
// import {
//     signInWithFacebook,
//     signInWithEmailAndPassword,
//     signUp,
//     signOut,
//     updateAccount
// } from '../../actions/authActions';
// import reducer, { initialState } from '../authReducer';
//
// describe('Testing Auth Reducer', () => {
//     it('handles signInWithFacebook action', () => {
//         expect(reducer(initialState, signInWithFacebook())).toMatchSnapshot();
//     });
//     it('handles signInWithEmailAndPassword action', () => {
//         expect(
//             reducer(
//                 initialState,
//                 signInWithEmailAndPassword({ email: 'email', password: 'password' })
//             )
//         ).toMatchSnapshot();
//     });
//     it('handles signUp action', () => {
//         expect(
//             reducer(
//                 initialState,
//                 signUp({ email: 'email', password: 'password', name: 'name', number: '1112223333' })
//             )
//         ).toMatchSnapshot();
//     });
//     it('handles signOut action', () => {
//         expect(reducer(initialState, signOut())).toMatchSnapshot();
//     });
//     it('handles updateAccount action', () => {
//         expect(
//             reducer(initialState, updateAccount({ name: 'name', number: '1112223333' }))
//         ).toMatchSnapshot();
//     });
// });
