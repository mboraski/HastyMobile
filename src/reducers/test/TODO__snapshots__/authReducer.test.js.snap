// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Auth Reducer handles signInWithEmailAndPassword action 1`] = `
Object {
  "error": null,
  "pending": false,
  "user": null,
}
`;

exports[`Testing Auth Reducer handles signInWithFacebook action 1`] = `
Object {
  "error": null,
  "pending": false,
  "user": null,
}
`;

exports[`Testing Auth Reducer handles signOut action 1`] = `
Object {
  "error": null,
  "pending": false,
  "user": null,
}
`;

exports[`Testing Auth Reducer handles signUp action 1`] = `
Object {
  "error": null,
  "pending": false,
  "user": null,
}
`;

exports[`Testing Auth Reducer handles updateAccount action 1`] = `
Object {
  "error": null,
  "pending": false,
  "user": null,
}
`;

exports[`Testing Auth Reducer reducer example 1`] = `Object {}`;
